debezium-testing/debezium-testing-openshift/src/test/java/io/debezium/testing/openshift/resources/ConnectorFactories.java

```diff
@@ -29,7 +29,7 @@ public ConnectorConfigBuilder mysql() {
                 .put("database.user", ConfigProperties.DATABASE_MYSQL_DBZ_USERNAME)
                 .put("database.password", ConfigProperties.DATABASE_MYSQL_DBZ_PASSWORD)
                 .put("database.server.name", "mysqldb") // this should be overwritten with unique name
-                .put("database.whitelist", "inventory") // might want to change
```

Comment:
@jcechace, can you check out this one. This could actually be a problem when using the current test suite with older versions as I think you do?

Reply:
Removing it certainly is not a good idea :). I'd say replace it with the new property and I will fix backward compatibility in separate PR.

Reply:
it's not removed @jcechace. Look at the full diff. Just Github diff characters threshold says the whole line is changed because it's essence is changed.

Reply:
Ah, I see. It should be all right then. Tis should be OK then. The question is whether are we going to backport it to downstream? If yes then I certainly do need to update the compatibility earlier. Anyway I'd solve this in separate PR

Reply:
@jpechane do you happen to know if this will be backported to downstream?

Reply:
@jpechane :point_up_2:

Reply:
@rk3rn3r, I don't think this is something we're going to back-port.  It's meant to be a part of 1.3+ afaik.